Makefile

```diff
@@ -34,6 +34,7 @@ $(call build-image,ocp-cluster-kube-controller-manager-operator,$(IMAGE_REGISTRY
 $(call add-bindata,v4.1.0,./bindata/v4.1.0/...,bindata,v411_00_assets,pkg/operator/v411_00_assets/bindata.go)
 
 test-e2e: GO_TEST_PACKAGES :=./test/e2e/...
+test-e2e: GO_TEST_FLAGS += -v
```

Comment:
Ditto as in the other PR, I don't think we need it here.

Reply:
I prefer to see what's going on rather than a hang, but I removed this change : )